src/main/docs/guide/config/customTypeConverter.adoc

```diff
@@ -19,7 +19,8 @@ This won't work by default, since there is no built-in conversion from `Map` to
 snippet::io.micronaut.docs.config.converters.MapToLocalDateConverter[tags="imports,class", indent=0]
 
 <1> The class implements api:core.convert.TypeConverter[] which has two generic arguments, the type you are converting from, and the type you are converting to
-<2> The implementation delegates to the default shared conversion service to convert the values from the Map used to create a `LocalDate`
-<3> If an exception occurs during binding, call `reject(..)` which propagates additional information to the container
+<2> The constructor injects a bean of type `ConversionService`, introduced in Micronaut 4, instead of making static calls to `ConversionService.SHARED` used in previous versions
+<3> The implementation delegates to the injected conversion service to convert the values from the Map used to create a `LocalDate`
+<4> If an exception occurs during binding, call `reject(..)` which propagates additional information to the container
 
-NOTE: It's possible to add a custom type converter into `ConversionService.SHARED` by registering it via the service loader.
+NOTE: New type converters can be added to api:core.convert.MutableConversionService[] retrieved from the bean context or by declaring a bean of type api:core.convert.TypeConverter[]. To register a type converter into `ConversionService.SHARED`, the registration needs to be done via the service loader; however, this is not recommended since `ConversionService.SHARED` is deprecated and will be removed in the next version of Micronaut Framework.
```

Comment:
```suggestion
NOTE: Register new type converters by declaring a bean of type api:core.convert.TypeConverter[]. `ConversionService.SHARED` is deprecated,  and it will be removed in a future version of the Micronaut Framework.
```